zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DbReader.scala

```diff
@@ -2,22 +2,52 @@ package com.broadinstitute.dsp
 package zombieMonitor
 
 import cats.effect.{Async, _}
-import doobie._
-import doobie.implicits._
 import fs2.Stream
 import DbReaderImplicits._
+import doobie._
+import doobie.implicits._
+import cats.implicits._
 
 trait DbReader[F[_]] {
   def getDisksToDeleteCandidate: Stream[F, Disk]
+  def getk8sClustersToDeleteCandidate: Stream[F, K8sClusterToScan]
+  def updateDiskStatus(id: Long): F[Unit]
+  def updateK8sClusterStatus(id: Long): F[Unit]
 }
 
 object DbReader {
   implicit def apply[F[_]](implicit ev: DbReader[F]): DbReader[F] = ev
 
+  val activeDisksQuery =
+    sql"""select id, googleProject, name from PERSISTENT_DISK where status != "Deleted" or status != "Error";
+        """.query[Disk]
+
+  val activeK8sClustersQuery =
+    sql"""select id, googleProject, location, clusterName from KUBERNETES_CLUSTER where status != "Deleted" or status != "ERROR";
```

Comment:
I think it should be capital `"DELETED"`

https://github.com/DataBiosphere/leonardo/blob/develop/core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala#L90

Reply:
hmm...was wondering how it worked in unit tests, and seems like it doesn't matter when I tried it out in MySQL client....will update it anyway tho

Reply:
Ah maybe case doesn't matter -- still good to be correct 👍